src/snapshot/tosvg.js

```diff
@@ -107,7 +107,7 @@ module.exports = function toSVG(gd, format) {
 
     // remove draglayer for Adobe Illustrator compatibility
     if(fullLayout._draggers) {
-        fullLayout._draggers.node().remove();
+        fullLayout._draggers.remove();
```

Comment:
man, we really got to get a Windows dev to wrap up https://github.com/plotly/plotly.js/issues/431 for us.